modules/auxiliary/gather/getprivate_ip_browsers

```diff
@@ -0,0 +1,83 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name'           => "Private IP Leakage to WebPage using WebRTC Function.",
+        'Description'    => %q(
+          This module exploits a vulnerability in browsers using well-known property of WebRTC .
```

Comment:
This seems kind of broad. Is it possible to give a bit more info?

Reply:
If you choose to not add more info, some references (links, edb, cve, etc.) would be very nice 👍